code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -591,15 +591,18 @@
 	reagent_state = LIQUID
 	color = "#21170E"
 	taste_description = "tea"
+	harmless = TRUE
```

Comment:
I don't think you need to specify this, `/datum/reagent/consumable` defaults to `harmless = TRUE` and you only need to specify if a reagent is not harmless.

Reply:
Damn it, this is what happens when I code at 3AM.
It does default to `TRUE`, but I'll leave this here anyway to avoid any possible confusion.